frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css

```diff
@@ -45,3 +45,7 @@
 .toolbarWrapper > * {
   pointer-events: auto;
 }
+
+:global([data-state='expanded']) + .main .toolbarWrapper {
+  display: none;
+}
```

Comment:
```suggestion
@media screen and (max-width: 768px) {
  :global([data-state='expanded']) + .main .toolbarWrapper {
    opacity: 0;
    visibility: hidden;
  }
}
```

Reply:
For devices with widths other than the Mobile version, I would like to see the toolbar displayed, so I would also like the following added to media query of the toolbarWrapper.

```
@media screen and (max-width: 768px) {
  .toolbarWrapper {
    transition: opacity 0.3s, visibility 0.3s;
    opacity: 1;
    visibility: visible;
    min-width: 354px;
  }
}
```

Setting min-width reduces layout wobble when scaling down.

Reply:
Thank you for your review and code🙏. I've implemented your suggestions, and the toolbar is now functioning much more smoothly✨. Thank you very much for your assistance.

fixed: https://github.com/liam-hq/liam/pull/359/commits/1c20fd1abe6b205c53a123c1b7487afd68e6532e